snippets/single-product.liquid

```diff
@@ -84,7 +84,8 @@
           {%- when 'product_description' -%}
             <section class='product-description margin-top'>{{ product.description }}</section>
           {%- when 'countdown_timer' %}
-            <div class='margin-top'>
+            <div class='countdown-block margin-top'>
```

Comment:
can you update margin-top to `mt-3` or `mt-4` we already have this utility

Reply:
didn't want to use those classes because we need to delete the file that has tailwind CSS classes

Reply:
it's not a tailwind CSS it's just a css file with a prebuild css code to make the way easier

<img width="541" alt="Screen Shot 2023-03-29 at 13 18 50" src="https://user-images.githubusercontent.com/108662497/228550735-af275bda-075e-43e0-b7ed-fae44702bc67.png">

Reply:
so no need to add .margin-top we already have some premade classes
<img width="380" alt="Screen Shot 2023-03-29 at 13 22 08" src="https://user-images.githubusercontent.com/108662497/228552597-abf453ac-2f61-4c00-a73e-92369e996c84.png">

Reply:
yes I know thats not tailwind class but it is similar to tailwind and this file isn't on our SCSS file that's make it harder to know what kind of CSS class we can use

Reply:
I agree, but instead of using `.margin-top` or `.mt-4`, try to add margin-top to the `.countdown-block` class

example
```
.countdown-block {
    margin-top: 20px;
    ...
    ...
}
```

and use it just like this
```
  {%- when 'countdown_timer' %}
          <div class='countdown-block'>
```

Reply:
i just checked that and I'm curious if you think this is a good approach to writing the classes 🤔

<img width="1009" alt="Screen Shot 2023-03-29 at 13 48 38" src="https://user-images.githubusercontent.com/108662497/228559195-ee0bd48a-99b2-4df4-b128-bdd88679186b.png">

Reply:
class name updated to `top-spacing`  I need this class so that's why I have create it for putting it in multiple elements